bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/module/provider/AnnotationActionModuleTypeHelper.java

```diff
@@ -61,6 +69,17 @@ public class AnnotationActionModuleTypeHelper {
     private static final String SELECT_THING_LABEL = "Select Thing";
     public static final String CONFIG_PARAM = "config";
 
+    private final ActionInputsHelper actionInputsHelper;
+
+    @Activate
+    public AnnotationActionModuleTypeHelper(final @Reference ActionInputsHelper actionInputsHelper) {
+        this.actionInputsHelper = actionInputsHelper;
+    }
+
+    @Deactivate
+    protected void deactivate() {
+    }
```

Comment:
If you do nothing, you don't need a `deactivate` method.

Reply:
I added this thinking it could be the cause of the feature resolution to sue. But it was certainly rather the missing service parameter in Component annotation.

Reply:
Removed